config/sync/field.field.rdf_entity.collection.field_ar_moderation.yml

```diff
@@ -11,7 +11,7 @@ id: rdf_entity.collection.field_ar_moderation
 field_name: field_ar_moderation
 entity_type: rdf_entity
 bundle: collection
-label: Moderation
+label: Moderated
```

Comment:
Ah I might be mistaken in my comment above, I possibly mixed up my screenshots. I agree that this should be "Moderated" and not "Moderation".